### DISCUSSION TITLE
Mettre à jour le ggodc pour les statistiques de classes

### DISCUSSION THREAD

**Discussion by Stokiwi** [Crownicles/Crownicles#1766] [Nouvelles idées]
Je voulais vous dire que sur le ggdoc qui permet de connaitre les statistiques de classes selon notre rang, la classe "Mage mystique" est absente, et je dois avouer que je souhaitais me renseigner un peu + sur cette classe, pensez-vous que le ggdoc aura une MAJ ? Merciii

**Comment by BastLast** [comment]
Ah oueeee bonne idée ! Faut retrouver qui l'avais fait à la base...

**Reply by Stokiwi** [reply]
Il me semble que c'est quelqu'un qui s'appelle Charette sur Discord si je me souviens de ce que j'ai lu dans le guide du joueur

**Reply by BastLast** [reply]
Faut le retrouver parce que jai pas la. Main sur le doc :/

**Reply by BastLast** [reply]
Ou alors jsp si on peut faire une copie
